Project-1/README.md

```diff
@@ -63,60 +68,256 @@ T	T	F	T	T
+
   ```(a) (A→B)→C
   (b) (A→(B∨C))∨(C→¬A)
   (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
-``` 
+
+``` (a) (A ∨ C) ∧ (¬B ∨ C)
+(b) A ∨ ¬B Tautology should have form of p v ~p
+(c) A v ~A Tautology  
+
+used CNF calculator https://www.artima.com/cs/cnf.html & https://www.erpelstolz.at/gateway/formular-uk-zentral.html
+
 
 3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
+
   ```(a)  B(x,y) that says that x is a brother of y
   (b)  A(x,y) that says that x is an aunt of y
   (c)  C(x,y) that says that x and y are cousins 
   (d)  O(x) that says that x is an only child
   (e)  T(x) that says that x has exactly two brothers 
-```
+
+```(a) ∃x∃y∃z(P(z,y)∧P(z,x)∧(x≠y)∧(z≠x)∧(z≠y)∧¬Fx) - There is some x and some y and some z such that z is the parent of y and z is the parent x and x is not female
+(a) A brother is an individual who identifies as male and has at least one sibling that is, they share at least one parent. Half-siblings and step-siblings can be brothers. In the example above, x would be an individual who shares at least one parent with y and x would identify as male.
+
+(b) ∃w∃x∃y∃z(P(z,y)∧P(w,z)∧P(w,x)∧Fx) - There is some w and some x and some y and some z such that z is the parent of y and w is the parent of z and w is the parent of x and x is female
+(b) An Aunt is a female identifying sibling of a parent. In the example above x would be an Aunt of y so long as x is the child of y's sibling
+
+(c) ∃v∃w∃x∃y∃z(P(v,w)∧P(v,z)∧P(w,x)∧P(z,y)) - There is some v and some w and some x and some y and some z such that v is the parent of w and v is the parent of z and w is the parent of x and z is the parent of y
+(c) A cousin is a relationship between two individuals whose parents are siblings. Such that you have a cousin in the child of your aunt
+
+(d) ∃x∃y∀z(P(y,z)→z=x) - There is some x and some y such that for all z if y is the parent of z then z and x are identical 
+(d) An only child is an individual who was raised in a house primarily without siblings. Individuals who have step or half siblings but were raised pimarily alone are only children. 
+
+(e) ∃v∃w∃x∃y∀z(P(v,x)∧P(v,w)∧¬Fw∧P(v,y)∧¬Fy∧∀z((P(v,z)∧¬Fz)→((z=x)∨(z=w)∨(z=y)))) - There is some v and some w and some x and some y such that for all z if v is the parent of x and v is the parent of w and w is not female and v is the parent of y and y is not female and for all instances of z v is a parent of z and z is not female ten z is identical with w or z is identical with w or z is identical with y
+(e) An individual has exactly two brothers if and only if they have two siblings who identify as male.
 
```

Comment:
I guess you were in the same group with Karl today so you probably don't need my comments on these, otherwise let me know

Reply:
I think I resolved this